CoEpi/ble/Peripheral.swift

```diff
@@ -1,33 +1,43 @@
-import Foundation
 import CoreBluetooth
+import Dip
+import Foundation
 import os.log
 import RxRelay
 
+struct Contact2 {
+    let theirData: Data?
```

Comment:
The logic for using Data type is... ?

Reply:
The point is to be able to pass the CEN data to consumers of the BT code, but not have the BT code know that it's CEN data/know what it is. So the BT code can just pass it out as Data without knowing what data it is. That's what this comment was about:
`next step would be to make Peripheral generic on theirData and ourData`

So the consumer can know that it's actually CENs but the Peripheral code just sees it as data